src/features/vision.feature

```diff
@@ -3,4 +3,38 @@ Feature: Test for Vision page
   Scenario: Open Vision Insurance
     Given the user is on the Vision page
     When the Vision Insurance product page is opened
-    Then the main Vision title and description should be correct
\ No newline at end of file
+    Then the main Vision title and description should be correct
+    And correct icon is displayed next to the Vision title
+    And the title and description of What is Vision Insurance? should be correct
+
+  Scenario: Breadcrumb links testing
```

Comment:
Scenario could be named better, for example:
Scenario: Check that breadcrumb links are correct and navigate user to correct pages

Reply:
Scenario name changed, and scenario was updated.